### PR TITLE
Updated booking status

### PR DESCRIPTION
**Pull request by margeol90**
I've listed the bookings on tabs - but has basic design. I found the html code online, so needs a little bit of research on how to change its CSS.

image
image

### CODE REVIEW

**Comment by Jide-Muritala** [comment]
Well done Margarita! This really looks great with the tabs.

File: app/models/office.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class Office < ApplicationRecord
   has_one_attached :photo
   belongs_to :user
   has_many :bookings, dependent: :destroy
-  has_many :office_facilities
+  has_many :office_facilities, dependent: :destroy
   has_many :facilities, through: :office_facilities
   # has_many :reviews, through: :bookings
```
Comment (Jide-Muritala) [inline app/models/office.rb:9]: What is the difference between facilities and office_facilites?

File: app/views/bookings/_form.html.erb [hunk 2]
```diff
@@ -1,6 +1,6 @@
-<%= simple_form_for [@office, @booking], data: { controller: 'flatpickr' } do |f| %>
-  <%= f.input :start_date, as: :string, input_html: { data: { flatpickr_target: "startDate" } } %>
-  <%= f.input :end_date, as: :string, input_html: { data: { flatpickr_target: "endDate" }, id: 'end_date' } %>
-  <%= f.submit class: "btn-confirm" %>
+<%= simple_form_for @booking, data: { controller: 'flatpickr',  unavailable_dates: @office.unavailable_dates.to_json }, method: :patch, id: "edit_booking" do |f| %>
```
Comment (Jide-Muritala) [inline app/views/bookings/_form.html.erb:1]: Hope after specifying the method here as :patch and id as edit_booking, the calendar still works when creating a totally new booking (where method is :create)

File: app/models/office.rb (hunk 1 above)
Comment (margeol90) [reply app/models/office.rb:9]: office_facilities is the join table between an office and a facility!

image

File: app/views/bookings/_form.html.erb (hunk 2 above)
Comment (margeol90) [reply app/views/bookings/_form.html.erb:1]: It should, as this _form is called only on the edit.html.erb, whereas the new booking form is on the show page.

Comment (margeol90) [reply app/views/bookings/_form.html.erb:1]: I think it works but If it doesn't, we know where to look!

File: app/models/office.rb (hunk 1 above)
Comment (Jide-Muritala) [reply app/models/office.rb:9]: Noted.

**Pull request merged**